client/src/store/new-request-module.ts

```diff
@@ -2675,7 +2675,7 @@ export class NewRequestModule extends VuexModule {
       name = sanitizeName(this.name)
     }
     ['entity_type_cd', 'request_action_cd', 'location'].forEach(field => {
-      if (this[field] === 'INFO') {
+      if (!this[field] || this[field] === 'INFO') {
```

Comment:
A comment describing what the code is doing here would be helpful. Eg, why check for empty?

Reply:
sure I can add something